### PR TITLE
Add SPM support.

### LINKED ISSUE #9: SPM support
Seems this is the one adequate Swift library for certificate generation.
Please add Swift Package Manager support

### PR DESCRIPTION
**Pull request by skogland**
- Replace Cocoapods with SPM for dependencies
- Add Package.swift
- Fixes an issue in DEREncoding.swift where the validFrom/validTo dates on the certificate not be correct for devices using a 12h date format by explicitly setting the locale to "nb". I think any locale with a 24h time format would suffice. The certificate dates would be about 100 years in the future. I had trouble reproducing the bug in the tests, it would be great to have it covered.

### CODE REVIEW

File: Package.resolved [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "repositoryURL": "https://github.com/iosdevzone/IDZSwiftCommonCrypto",
+        "state": {
+          "branch": null,
+          "revision": "d824371e670bd57eb456bbc41139b4997f7207b8",
+          "version": "0.13.1"
+        }
+      },
+      {
+        "package": "SwiftBytes",
+        "repositoryURL": "https://github.com/skogland/swift-bytes.git",
```
Comment (skogland) [inline Package.resolved:15]: The swift-bytes repo needs a version tag on the release with SPM-support to be included. This points to my fork with a version tag. Update the URL when the master is tagged.

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        .library(
+            name: "SelfSignedCert",
+            targets: ["SelfSignedCert"]
+        )
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(url: "https://github.com/iosdevzone/IDZSwiftCommonCrypto", from: "0.13.1"),
+        .package(name: "SecurityExtensions", url: "https://github.com/skogland/swift-SecurityExtensions", from: "5.0.0"),
+        .package(name: "SwiftBytes", url: "https://github.com/skogland/swift-bytes.git", from: "0.8.0")
```
Comment (skogland) [inline Package.swift]: The swift-bytes repo needs a version tag on the release with SPM-support to be included. This points to my fork with a version tag. Update the URL when the master is tagged. See https://github.com/dapperstout/swift-bytes/issues/5

File: Package.swift [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "SelfSignedCert",
+            targets: ["SelfSignedCert"]
+        )
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(url: "https://github.com/iosdevzone/IDZSwiftCommonCrypto", from: "0.13.1"),
+        .package(name: "SecurityExtensions", url: "https://github.com/skogland/swift-SecurityExtensions", from: "5.0.0"),
```
Comment (skogland) [inline Package.swift]: Update url when https://github.com/svdo/swift-SecurityExtensions/pull/4 is merged.

**Comment by svdo** [comment]
Wow you're on a roll Eirik, I'm super happy with this! :) The tag 0.8.0 on swift-bytes has come through, so that can be changed as well now I guess. Also, I merged your pull request on swift-SecurityExtensions.

Is there a particular reason for completely removing CocoaPods support? If possible, I'd like to keep that for the moment.

**Comment by skogland** [comment]
I don't belive it was a reason for removing the podspec other than that I wasn't going to use it and I wasn't going to test that it worked after moving to SPM for dependencies. I believe it should be fine adding it back!

**Pull request merged**